### PR TITLE
updates status resource

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/1918.

### CODE REVIEW

File: service/controller/v14/cluster_resource_set.go [hunk 1]
```diff
@@ -371,24 +370,42 @@ func NewClusterResourceSet(config ClusterResourceSetConfig) (*controller.Resourc
...
+	//
+	//	statusResource, err = statusresource.New(c)
+	//	if err != nil {
+	//		return nil, microerror.Mask(err)
+	//	}
+	//}
 
 	resources := []controller.Resource{
-		statusResource,
+		//statusResource,
```
Comment (rossf7) [inline service/controller/v14/cluster_resource_set.go]: I'm playing catch up but I miss some context on why the statusResource is being disabled. I think a comment would be useful.

Comment (kopiczko) [reply service/controller/v14/cluster_resource_set.go]: +1 interesting :) Also happened after my review.

Comment (xh3b4sd) [reply service/controller/v14/cluster_resource_set.go]: I did this like ten minutes ago. I will provide information when I am ready.

Comment (xh3b4sd) [reply service/controller/v14/cluster_resource_set.go]: Please have another look. I added a TODO comment and an issue for it. See https://github.com/giantswarm/giantswarm/issues/3822.

Comment (rossf7) [reply service/controller/v14/cluster_resource_set.go]: I don't see the comment. Can you check?

Comment (xh3b4sd) [reply service/controller/v14/cluster_resource_set.go]: Oh yeah, when I push the code you see the comment. :D Please check now.

**Comment by xh3b4sd** [comment]
e2e tests passed already. Going.

**Pull request merged**